portfolio/src/main/webapp/script.js

```diff
@@ -44,16 +44,19 @@ async function getMessages(){
   const response = await fetch('/data');
   const json = await response.json();
   const container = document.getElementById('message-container');
-  container.appendChild(createListElement(json[0]));
-  container.appendChild(createListElement(json[1]));
-  container.appendChild(createListElement(json[2]));
+  json.forEach(function(comment){
```

Comment:
ES6 introduced the concept of arrow functions, which would be a little bit shorter (and more JavaScript-y). Something like this:

```
json.forEach((comment) => {
  // your code here
});
```

This is pretty much just syntactic sugar in this case though, so it's up to you which you like best.